docs/CHANGELOG.asciidoc

```diff
@@ -68,6 +68,437 @@ Review important information about the {kib} 7.x releases.
 
 coming::[7.15.0]
 
+For information about the {kib} 7.15.0 release, review the following information.
+
+[float]
+[[known-issue-7.15.0]]
+=== Known issue
+
+.Upgrade Elastic Agents to use Osquery Manager integration
+[%collapsible]
+====
+*Details* +
+You must upgrade your Elastic Agents to the latest version to use the Osquery Manager integration.
+
+*Impact* +
+To upgrade, refer to {fleet-guide}/upgrade-elastic-agent.html[Upgrade Elastic Agent].
+====
+
+.APM: Metrics-powered throughput is incorrect
+[%collapsible]
+====
+*Details* +
+In some cases, users who have enabled APM aggregated transactions (metrics-powered UI), will see throughput incorrectly stuck on 1tpm.
+*Impact* +
```

Comment:
Sorry, just noticed in the preview that a space needs to exist for these to render correctly. 
```suggestion

*Impact* +
```